charts/hawkbit-update-server/values.yaml

```diff
@@ -31,21 +31,19 @@ ingress:
     # nginx.ingress.kubernetes.io/proxy-body-size: 200m
   paths: []
   hosts:
-    - chart-example.local
+    - hawkbit-update-server.local
   tls: []
-  #  - secretName: chart-example-tls
+  #  - secretName: hawkbit-tls
   #    hosts:
-  #      - chart-example.local
+  #      - hawkbit-update-server.local
 
 # env vars for configuration
 env:
   springDatasourceHost: "hawkbit-update-server-mysql"
-
   springDatasourceDb: "hawkbit"
   springRabbitmqHost: "hawkbit-update-server-rabbitmq"
   springRabbitmqUsername: "hawkbit"
   springRabbitmqPassword: "hawkbit"
-  javaOptions: "-Xms512m -Xmx512m"
```

Comment:
I would prefer to have javaOptions still in place (at least commented), as hawkbit is running w/ Java 1.8 the JVM is not aware of available resources.
In fact javaOptions should be a bit lower than pod mem-resoure configuration

Reply:
We already have it here: https://github.com/kiwigrid/helm-charts/blob/master/charts/hawkbit-update-server/values.yaml#L51-L52

Removed it as it looked like a duplicate.